frameworks/PHP/symfony/symfony.dockerfile

```diff
@@ -5,30 +5,20 @@ ARG DEBIAN_FRONTEND=noninteractive
 RUN apt-get update -yqq && apt-get install -yqq software-properties-common > /dev/null
 RUN LC_ALL=C.UTF-8 add-apt-repository ppa:ondrej/php
 RUN apt-get update -yqq > /dev/null && \
-    apt-get install -yqq nginx git unzip php7.3 php7.3-common php7.3-cli php7.3-fpm php7.3-mysql  > /dev/null
-RUN apt-get install php7.3-xml  > /dev/null
+    apt-get install -yqq nginx git unzip php7.3 php7.3-common php7.3-cli php7.3-fpm php7.3-mysql > /dev/null
+RUN apt-get install -yqq php7.3-curl php7.3-intl php7.3-mbstring php7.3-opcache php7.3-xml > /dev/null
```

Comment:
Some extensions are not necessary in this bench.
Using exactly :
`RUN apt-get install -yqq php7.3-mbstring php7.3-xml  > /dev/null`

It will also use the docker cache, as this line is used in others frameworks.

Reply:
also curl made quite a speedup when installing composer packages on my machine, maybe keep it in a separate line of `ext-intl`?

Reply:
Do you think that is it necessary the php-intl ?

This extension is only used for localization.
This bench is only in English and without any localization.

If the framework install the polifill without being used, for me that is an architectural fail.

Reply:
also i've found interesting problem
i was tripped by docker cache as i couldn't just add 
`RUN apt-get install -yqq php7.3-intl > /dev/null` as the third line - package lists/remote repo has been changed (php was updated to 7.3.8 and this command tried to install intl 7.3.7 which was no longer available)

i've could add `apt-get update` before install but that just looks like a hack.

any suggestions?

Reply:
I have a PR ready for these changes.
But if I send it, it will break your PR.

Reply:
The solution is to join the lines when it is posible.

Reply:
i'll drop `intl` (it's used for localization of routes and domains)
for your pr please consider adding `curl` it really speeds up composer

don't worry about this pr, i'll gladly merge any changes.